[doc][kuberay] state `rayStartParams` is optional starting with KubeRay 1.4.0 (#53943)
ray-project/ray: doc/source/cluster/kubernetes/user-guides/config.md

## Ray Start Parameters
The ``rayStartParams`` field of each group spec is a string-string map of arguments to the Ray
container’s `ray start` entrypoint. For the full list of arguments, refer to
the documentation for {ref}`ray start <ray-start-doc>`. The RayCluster K8s custom resource

> definition (CRD) in KubeRay versions before 1.4.0 requires this field to exist, but the value can be
> an empty map. Since KubeRay 1.4.0, ``rayStartParams`` is optional.

Reviewer: ```suggestion
definition (CRD) in KubeRay versions before 1.4.0 required this field to exist, but the value could be
an empty map. As of KubeRay 1.4.0, ``rayStartParams`` is optional.
```
